srcpkgs/python3-hjson/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'python3-hjson'
+pkgname=python3-hjson
+version=3.0.1
+revision=1
+archs=noarch
+wrksrc="hjson-${version}"
+build_style=python3-module
+hostmakedepends="unzip python3-setuptools"
+depends="python3"
+short_desc="User interface for JSON"
+maintainer="RinsedSloth <afernandezh@protonmail.com>"
+license="MIT"
+homepage="https://github.com/hjson/hjson-py"
+distfiles="${PYPI_SITE}/h/hjson/hjson-${version}.tar.gz"
+checksum=1d1727faa6aaef2973921877125a3ab7c5f6d34b93233179d01770f41fab51f9
+
+pkg_install() {
```

Comment:
this whole function shouldn't be necessary, there are no subpackages

Reply:
Oh right, forgot to delete after removing the subpackage. Thank you!